data_wizard/serializers.py

```diff
@@ -24,8 +24,8 @@ def to_internal_value(self, data):
         except ContentType.DoesNotExist:
             self.fail('does_not_exist', app_label=app_label, model=model)
 
-    def to_representation(self, content_type_id):
-        ct = ContentType.objects.get(pk=content_type_id)
+    def to_representation(self, content_type):
+        ct = ContentType.objects.get(pk=content_type.pk)
```

Comment:
Alternatively, rather than redoing the lookup, we can just immediately:

```
def to_representation(self, content_type):
    return '%s.%s' % (content_type.app_label, content_type.model)
```